base/compiler/tfuncs.jl

```diff
@@ -461,7 +461,7 @@ function sizeof_nothrow(@nospecialize(x))
         return sizeof_nothrow(rewrap_unionall(xu.a, x)) &&
                sizeof_nothrow(rewrap_unionall(xu.b, x))
     end
-    t, exact, isconcrete = instanceof_tfunc(x)
+    t, exact, isconcrete = instanceof_tfunc(x, false)
```

Comment:
We probably want to convert all `instanceof_tfunc(x, false)` to `instanceof_tfunc(x)`? I'm fine with the current state if you prefer explicitly specifying the value of the `astag` argument.

Reply:
I like the more explicit nature of it